Test_oM/Error.cs

```diff
@@ -34,5 +34,7 @@ public class Error : BHoMObject
         public ErrorLevel Level { get; set; }
         public string Message { get; set; }
         public Location Location { get; set; }
+
+        private double DeliberateFail { get; set; }
```

Comment:
This is appropriately failing the test. Whilst it is demonstrating the test works it needs fixing now.

Reply:
Howzat :wink: